frontend/src/router/index.js

```diff
@@ -174,11 +176,24 @@ const router = createRouter({
           name: 'declare-submit',
           component: DeclareSubmitView,
         },
+        {
+          path: ':applicationGuid/confirmation',
+          name: 'application-confirmation',
+          component: ApplicationConfirmationView,
+        },
       ],
       meta: {
         requiresAuth: true,
       },
     },
+    {
+      path: '/supp-applications',
```

Comment:
I think the terminology within the portal is Supplementary Allowance. So maybe /supp-allowances and SupplementaryAllowanceView?

Reply:
it was fixed.